tests/jballerina-unit-test/src/test/resources/test-src/statements/arrays/sealed_array.bal

```diff
@@ -15,6 +15,8 @@
 // under the License.
 import ballerina/lang.'int;
 
+const constLength = 2;
```

Comment:
Can we also add positive tests for module level, anon functions, etc. array definitions with constants?

Reply:
Need to also include positive and negative tests for scenarios where the constant is defined after the array in the source.

For example, the following should work.

```ballerina
int[TWO] x = [1, 2];

const TWO = 2;
```

and the following should result in an error.

```ballerina
int[ONE] x = [1, 2];

const ONE = 1;
```